torch/csrc/jit/script/module.cpp

```diff
@@ -30,14 +30,14 @@ static FunctionSchema defaultSchemaFor(const Method& method) {
 }
 
 
-const FunctionSchema& Method::getSchema() const {
+TORCH_EXPORT const FunctionSchema& Method::getSchema() const {
```

Comment:
I think that we should add these annotations in the function definitions, not implementations.